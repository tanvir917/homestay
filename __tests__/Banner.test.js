import React from 'react'
import TestRenderer from 'react-test-renderer';
import Banner from '../src/Components/Banner/Banner';


it('renders correctly', () => {
  const tree = TestRenderer
    .create(<Banner />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});