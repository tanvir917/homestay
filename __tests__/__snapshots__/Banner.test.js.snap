// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="banner"
>
  <h1 />
  <div />
  <p />
</div>
`;
